src/routes/organizers.jsx

```diff
@@ -247,6 +255,8 @@ export default function Organizers() {
             </div>
 
             <p class="text-center mt-6">
+              Co-organised with OST Eastern Switzerland University of Applied Sciences.
```

Comment:
Link as per @fmehta's suggestion https://github.com/nixcon/2025.nixcon.org/pull/21#discussion_r2068613871
> May I hake the following suggestion for the footer sentence:
> 
> "Co-organised with [OST Eastern Switzerland University of Applied Sciences](https://www.ost.ch/de/forschung-und-dienstleistungen/informatik/ifs-institut-fuer-software)"

(... and leave out the period, as this is not a complete sentence.)

```suggestion
              Co-organised with <a href="https://www.ost.ch/de/forschung-und-dienstleistungen/informatik/ifs-institut-fuer-software">OST Eastern Switzerland University of Applied Sciences</a>
```